src/backend/api/reservations.js

```diff
@@ -0,0 +1,82 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (req, res) => {
+  try {
+    const reservations = await knex("reservation");
+    res.json(reservations);
+  } catch (err) {
+    throw new Error("something went wrong on fetching reservations");
+  }
+});
+router.post("/", async (req, res) => {
+  try {
+    const reservationToAdd = {
+      number_of_guests: req.body.number_of_guests,
+      contact_phonenumber: req.body.contact_phonenumber,
+      contact_name: req.body.contact_name,
+      contact_email: req.body.contact_email,
+      meal_id: req.body.meal_id,
+      created_date: req.body.created_date,
+    };
+    const result= await knex("reservation").insert(reservationToAdd);
+    res.json(result);
+  } catch (error) {
+    throw new Error("something went wrong with adding meal");
```

Comment:
This error message seems to be wrong. looks like it was just copy-pasted (should probably mention reservations not meals)